semgrep_agent/dontfixme.py

```diff
@@ -0,0 +1,3 @@
+import pickle
+
+pickle.dumps("hi")
```

Comment:
Did you mean to commit these?

Reply:
I was using those files just to test the changes the PR introduced. It didn't work as I expected, was going to figure out why later.